lua/telescope/pickers/layout_strategies.lua

```diff
@@ -218,7 +226,12 @@ end
     +-----------------+
 --]]
 layout_strategies.vertical = function(self, max_columns, max_lines)
-  local layout_config = self.layout_config or {}
+  local layout_config = validate_layout_config(self.layout_config or {}, {
+    width_padding = "How many cells to pad the width",
+    height_padding = "How many cells to pad the height",
+    preview_width = "(Resolvable): Determine preview width",
```

Comment:
Shouldn't this be `preview_height` because it's vertical?

Reply:
Yes, that is fixed now